### PR TITLE
maven to gradle

### PR DESCRIPTION
**Pull request by ankit0706**
A small java maven project which is converted into gradle

### CODE REVIEW

File: gradle/maven-to-gradle/src/main/java/com/sample/javacode/DisplayTime.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+public class DisplayTime {
+
+	public static void main(String[] args) {
+		Date date = new java.util.Date();
+		System.out.println(date);
+		String gmt = date.toString();
+		boolean flag = StringUtils.isEmpty(gmt);
+		if(flag) {
+			System.out.println("No time found");
+		}else {
```
Comment (andrzejsydor) [inline gradle/maven-to-gradle/src/main/java/com/sample/javacode/DisplayTime.java:16]: please format code

Comment (ankit0706) [reply gradle/maven-to-gradle/src/main/java/com/sample/javacode/DisplayTime.java]: I have imported the eclipse formatter into my IDE and formatted the code. Please let me know if it's fine now.

Comment (JavierPeris) [reply gradle/maven-to-gradle/src/main/java/com/sample/javacode/DisplayTime.java]: The format is correct now.

File: gradle/maven-to-gradle/pom.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (JavierPeris) [inline gradle/maven-to-gradle/pom.xml:1]: This file is using tabs, please convert to spaces.

Comment (ankit0706) [reply gradle/maven-to-gradle/pom.xml:1]: I tried to fix it. Please let me know if it is fine now.

Comment (JavierPeris) [reply gradle/maven-to-gradle/pom.xml:1]: Looks good!

**Pull request merged**